src/globals/objects/_main-wrapper.scss

```diff
@@ -1,22 +1,38 @@
-// Example usage:
+// Example usage with Breadcrumbs, phase banners, back links:
 // <div class="govuk-o-width-container">
 //   <!-- Breadcrumbs, phase banners, back links are placed in here. -->
 //   <div class="govuk-o-main-wrapper">
 //       <!-- Wrapper for the main content of your page which applies padding
 //            to the top / bottom -->
 //   </div>
 // </div>
+//
+// Example usage without Breadcrumbs, phase banners, back links:
+// <div class="govuk-o-width-container">
+//   <div class="govuk-o-main-wrapper--l">
+//       <!-- Wrapper for the main content of your page which applies padding
+//            to the top / bottom -->
+//   </div>
+// </div>
+
 
 @mixin govuk-main-wrapper {
-  padding: $govuk-spacing-scale-3 0;
+  @include govuk-responsive-padding($govuk-spacing-responsive-6, "top");
+  @include govuk-responsive-padding($govuk-spacing-responsive-6, "bottom");
+}
 
-  @include mq($from: tablet) {
-    padding: $govuk-spacing-scale-6 0;
-  }
+// Use govuk-main-wrapper--l when you page does not have Breadcrumbs, phase banners or back links
+@mixin govuk-main-wrapper--l {
+  @include govuk-responsive-padding($govuk-spacing-responsive-8, "top");
+  @include govuk-responsive-padding($govuk-spacing-responsive-8, "bottom");
```

Comment:
Not sure why the presence (or absence) of breadcrumbs / back links would affect the padding at the bottom of the page – should this still be `responsive-6`?